app/helpers.php

```diff
@@ -1509,7 +1509,7 @@ function get_params($input, $namespace, $keys, $options = [])
 
     $params = [];
 
-    if (is_array($input) || ($input instanceof ArrayAccess)) {
+    if (is_iterable($input)) {
```

Comment:
`Generator`s are `iterable` but can't be accessed by array syntax used by `array_get` (which uses `is_array` and `ArrayAccess` for the check, anyway)